src/components/ui/QuantitySelector/QuantitySelector.tsx

```diff
@@ -7,15 +7,17 @@ import './quantity-selector.scss'
 interface QuantitySelectorProps {
   max: number
   min: number
+  initial?: number
   disabled: boolean
 }
 
 export function QuantitySelector({
   max,
   min,
+  initial,
   disabled,
 }: QuantitySelectorProps) {
-  const [quantity, setQuantity] = useState<number>(min)
+  const [quantity, setQuantity] = useState<number>(initial ?? min)
```

Comment:
I'm curious about this `??` operator

Reply:
@saranicoly, it is the [Nulllish Coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).

There is also [this good chapter](https://www.executeprogram.com/courses/everyday-typescript/lessons/nullish-coalescing) in the Execute Program: Everyday TypeScript.